conda/recipes/pynvjitlink/conda_build_config.yaml

```
@@ -0,0 +1,11 @@
+c_compiler_version:
+  - 11
+
+cxx_compiler_version:
+  - 11
+
+cuda_compiler:
+  - cuda-nvcc
+
+cuda_compiler_version:
+  - 12
```

Think we may need to fill in the minor version part so we can access that in the recipe. Perhaps there's a better solution than doing this

```suggestion
  - 12.2
```

---

Without this am getting errors like...

```
$ conda render conda/recipes/pynvjitlink
Error: Failed to render jinja template in /Users/jkirkham/Developer/rapidsai/pynvjitlink/conda/recipes/pynvjitlink/meta.yaml:
list object has no element 1
```

..., which I'm reading as this line failing when `cuda_compiler_version = "12"`

https://github.com/rapidsai/pynvjitlink/blob/cc57af8de24e7101f163d6f96931e05e9ab49ac3/conda/recipes/pynvjitlink/meta.yaml#L7

---

Just saw this, too -- fixed in b5d04c3.